src/main/java/eu/europeana/metis/sandbox/controller/advice/ControllerErrorHandler.java

```diff
@@ -4,16 +4,18 @@
 import eu.europeana.metis.sandbox.common.exception.NonRecoverableServiceException;
 import eu.europeana.metis.sandbox.common.exception.ServiceException;
 import eu.europeana.metis.sandbox.dto.ExceptionModelDto;
-import lombok.extern.slf4j.Slf4j;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.ControllerAdvice;
 import org.springframework.web.bind.annotation.ExceptionHandler;
 
 @ControllerAdvice
-@Slf4j
 public class ControllerErrorHandler {
 
+  private static final Logger log = LoggerFactory.getLogger(ControllerErrorHandler.class);
```

Comment:
We have the convention of using LOGGER as the name if this kind of field, same goes for the others

Reply:
Sure I will switch those